man/systemd.network.xml

```diff
@@ -547,6 +547,17 @@
             name servers limited to a specific link.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>DNSDefaultRoute=</varname></term>
+          <listitem>
+            <para>Take a boolean argument. If true, this link's configured DNS servers are used for resolving domain
+            names that do not match any link's configured <varname>Domains=</varname> setting. If false, this link's
+            configured DNS servers are never used for such domains, and are exclusively used for resolving names that
+            match at least one of the domains configured on this link. If not specified defaults to an automatic mode:
+            if any routing domains are configured on the link (see above) queries not matching any link's configured
```

Comment:
Please insert a comma after "(see above)".

Reply:
(i used your suggested rephrasing, below, which drops the see above anyway, so i guess this doesn't apply anymore)